### PR TITLE
[iOS] Predicate for live-photo filtering workaround

### PR DESCRIPTION
**Pull request by yu840915**
Use `NOT` instead of `!=` in live-photo filtering expression.
With original predicate, a lot of non-live-photo cannot bet fetched unexpectedly.

### CODE REVIEW

**Comment by AlexV525** [comment]
> With original predicate, a lot of non-live-photo cannot bet fetched unexpectedly.

Can you explain more with "cannot be fetched unexpectedly"?

**Comment by yu840915** [comment]
### Before:
On iOS, `PhotoManager.getAssetPathList(type: RequestType.image, filterOption: FilterOptionGroup(containsLivePhotos: false))`, returns number of albums much fewer than native **Photos** app has.

### After: 
`PhotoManager.getAssetPathList(type: RequestType.image, filterOption: FilterOptionGroup(containsLivePhotos: false))`, returns number albums matches native **Photos** app.

**Comment by AlexV525** [comment]
> returns number of albums much fewer than native **Photos** app has.

Why it's fewer before? What's the final predicate before and after applying the fix? How does it affect photo fetching? What kinds of photos are prevented to be fetched from the previous predicate?

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Improvements:
 
 - Correct `PermissionRequestOption` typo with a class type alias.
   Which also raised the Dart SDK constraint to `2.13.0`.
+- The original predicate for filtering live-photo cannot be executed correctly and unexpectedly rejects photos with media subtype `PHAssetMediaSubtypeNone`. Changing it from `( ( mediaSubtype & 8 ) != 8 )` to `NOT ( ( mediaSubtype & 8 ) == 8` works correctly.
```
Comment (AlexV525) [inline CHANGELOG.md:13]: We don't need a detailed explanation in the CHANGELOG. It should be placed around the updated code instead.

**Review by AlexV525 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (yu840915) [reply CHANGELOG.md:13]: Agree, how about this?
```suggestion
- The original predicate for filtering live-photo cannot be executed correctly and unexpectedly rejects photos with media subtype `PHAssetMediaSubtypeNone`. Changing it from `( ( mediaSubtype & 8 ) != 8 )` to `NOT ( ( mediaSubtype & 8 ) == 8` works correctly.
- Workaround for live-photo filtering bug in PhotoAsset framework on iOS.
```

Comment (AlexV525) [reply CHANGELOG.md:13]:
```suggestion
- Improve Live-Photos filtering.
```

**Pull request merged**